src/components/Page/components/Header/Header.scss

```diff
@@ -142,11 +155,46 @@ $action-menu-rollup-computed-width: rem(40px);
   display: flex;
   align-content: flex-start;
   align-items: center;
+
+  .newDesignLanguage & {
+    align-items: flex-start;
+    flex: 1 1 auto;
+
+    @include page-content-when-not-partially-condensed {
+      flex-basis: 50%;
```

Comment:
Wouldn't mind pairing with someone on this to see if we can remove flex-basis altogether and make these containers a little more fluid